examples/i3module/service_control.py

```diff
@@ -41,10 +42,11 @@ def build_service_conf(hitnet, chargenet, theta_prior, t_prior):
         "boundary_guard": {
             "file": theta_prior,
             "param_limits": DEFAULT_SEARCH_LIMITS,
-            "bg_lim": -10,
+            "bg_lim": 0,
             "invalid_llh": 1e9,
-            "prior": True,
+            "prior": False,
```

Comment:
is this turning the prior off?

Reply:
Yes, in this configuration it's just used as boundary guard. If we have a theta prior that is a bit more tuned, we can turn it on again

Reply:
ok, but for the big collab. meeting reco we did use it as prior? I just want to avoid changing too many things at a time

Reply:
Yes, for the big reco we used it as prior